internal/crypto/crypto.go

```diff
@@ -6,6 +6,9 @@ import (
 	"golang.org/x/crypto/curve25519"
 )
 
+// NewKeyPair returns a pair of cryptographic keys that can later
+// be used for signature and verification, along with an error,
```

Comment:
Encryption!

Reply:
Sorry, I meant only encryption, or "encryption and decryption" if you want to be pedantic ^^